tests/test_landfill_sampler.py

```diff
@@ -11,19 +11,19 @@ def sample_document():
     return {
         'content': {"payload": {"foo": "bar"}},
         'meta': {
-            u'Content-Length': u'7094',
-            u'Date': u'Sun, 19 Aug 2018 15:08:00 GMT',
-            u'Host': u'incoming.telemetry.mozilla.org',
-            'Hostname': u'ip-1.1.1.1',
-            'Timestamp': 1534691279765301222L,
-            'Type': u'telemetry-raw',
-            u'User-Agent': u'pingsender/1.0',
-            u'X-Forwarded-For': u'127.0.0.1',
-            u'X-PingSender-Version': u'1.0',
-            u'args': u'v=4',
-            u'protocol': u'HTTP/1.1',
-            u'remote_addr': u'1.1.1.1',
-            u'uri': u'/submit/telemetry/doc-id/main/Firefox/61.0.2/release/20180807170231'
+            'Content-Length': '7094',
```

Comment:
removing the `u` is a functional change here for python 2, and a noop in python 3. why is it being changed?